### PR TITLE
feat(pagewizard): afterFooterContent new prop

### LINKED ISSUE #1717: [PageWizardStep] afterFooterContent new prop
### What package is this for?

- [x] React
- [ ] Angular

### Summary

New prop included for PageWizardStep to add an additional content at the end of the footer.

*Screenshots*
image
image

### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?
Yes, MAS 8.2 release on October 23rd

### Available extra resources

What resources do you have to assist this effort?
I'll work on it.

### PR DESCRIPTION
**Pull request by hectordavis**
Closes #1717

**Summary**
New prop `afterFooterContent` added to `PageWizardStep` 

image
image

**Change List (commits, features, bugs, etc)**

- feat(pagewizard): afterFooterContent new prop

**Acceptance Test (how to verify the PR)**

- Validate if PageWizardStep still works with and without the new `afterFooterContent` prop

### CODE REVIEW

**Review by cgirani — APPROVED** [review]
LGTM 👍

**Review by bjornalm — APPROVED** [review]
Looks good, just one minor thing it RTL mode where spacing is missing between the icon and the text:

image

File: src/components/PageWizard/PageWizardStep/PageWizardStep.jsx [hunk 1]
```diff
@@ -132,6 +137,12 @@ const PageWizardStep = ({
           {i18n.submit}
         </Button>
       )}
+      {afterFooterContent && (
+        <div
+          className={`${iotPrefix}--page-wizard--content--after-footer--content`}>
+          {afterFooterContent}
+        </div>
+      )}
```
Comment (tay1orjones) [inline src/components/PageWizard/PageWizardStep/PageWizardStep.jsx:145]: > Looks good, just one minor thing it RTL mode where spacing is missing between the icon and the text:
> 
> image

@bjornalm I believe that's just part of the story, the rendered node is fully customizable here

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**